src/Components/GameSession.js

```diff
@@ -21,34 +21,38 @@ export default function GameSession() {
   function handleTileClick(e) {
     let id = e.currentTarget.id;
     if (id !== null) {
-      const currentDauberedTiles = dauberedTiles;
-      if (currentDauberedTiles[id] !== true) {
-        currentDauberedTiles[id] = true;
+      dauberTile(id);
+    }
+  }
+
+  function dauberTile(id) {
+    const currentDauberedTiles = dauberedTiles;
+    if (currentDauberedTiles[id] !== true) {
+      currentDauberedTiles[id] = true;
+      if (id !== 12) {
+        // do not count FREE space
         let moveCount = moves;
         moveCount++;
-        setDauberedTiles(currentDauberedTiles);
         setMoves(moveCount);
       }
+      setDauberedTiles(currentDauberedTiles);
     }
   }
 
-  function dauberTile(id) {
-    setDauberedTiles((prev) => {
-      let modDauberedTiles = prev;
-      modDauberedTiles[id] = true;
-      return modDauberedTiles;
-    });
-  }
-
   function restartGame() {
-    setGamesStarted(gamesStarted + 1);
     setMoves(0);
     setDauberedTiles([]);
-    setGamesStarted(gamesStarted + 1);
+    let gamesStartedCount = gamesStarted;
+    gamesStartedCount++;
+    dauberTile(12);
+    setGamesStarted(gamesStartedCount);
   }
 
   useEffect(() => {
     setBingoed(checkForBingo(dauberedTiles, moves));
+    let currentDauberedTiles = dauberedTiles;
```

Comment:
This works but we are still re-daubering the center tile every time a new tile is clicked.

I have another suggestion that can hopefully be a once-and-for-all solution for the problem of daubering the center tile.

My suggestion is to pass a default array with index 12 set to true as the default value of dauberedTiles when declaring the state variables. Something like:

```
const defaultDauberedTiles = [];
defaultDauberedTiles[12] = true;
const [dauberedTiles, setDauberedTiles] = useState(defaultDauberedTiles);
```

We can then add `setDauberedTiles(defaultDauberedTiles);` to `restartGame()` and remove all of the other code in `GameSession` daubering the center tile. We also won't need to check for the free tile in the `dauberTile` method, since we won't be using `dauberTile` to dauber the center tile anymore.